time_weighted_average_price/attest_fn_call/main.go

```diff
@@ -173,10 +173,4 @@ func twap(inputPtr, secretPtr uint64) uint64 {
 	return WriteOutput(twap)
 }
 
-//export errorFunc
-func errorFunc(inputPtr, secretPtr uint64) uint64 {
-	err := errors.New("Expected error for testing")
-	return WriteError(err)
-}
-
```

Comment:
I had this error function in here to generate a `Result` with an error to then show in the TWAP onchain that we can revert a transaction - this was out of scope for this example